modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc

```diff
@@ -1,16 +1,26 @@
 = Multi-tenant configuration for a Bonita platform
-:description: (Deprecated). One single platform to admininstrate with multiple logical engines. Described what a Tenant is and how to setup Bonita with Multi-tenancy.
+:description: (Deprecated). One single platform to administrate with multiple logical engines. Described what a Tenant is and how to setup Bonita with Multi-tenancy.
 
 [WARNING]
 ====
+This feature was deprecated in Bonita 2021.1. This means that Multi-tenant architecture will continue to work in Bonita for existing Bonita subscription customers,
```

Comment:
I think there is no need to say 'in 2021.1', you are already on the 2021.1 documentation. 
It is either you are on a deprecated version or not, so just say 'it is deprecated' when it is the case. 

```suggestion
This feature is deprecated. +
This means that Multi-tenant architecture will continue to work in Bonita for existing Bonita subscription customers,
```